src/k8s/pkg/k8sd/app/hooks_remove.go

```diff
@@ -144,6 +143,11 @@ func (a *App) onPreRemove(ctx context.Context, s state.State, force bool) (rerr
 		if err := snaputil.StopControlPlaneServices(ctx, snap); err != nil {
 			log.Error(err, "Failed to stop control-plane services")
 		}
+
+		log.Info("Stopping k8s-dqlite")
+		if err := snaputil.StopK8sDqliteServices(ctx, snap); err != nil {
```

Comment:
We have a similar implementation here: https://github.com/canonical/k8s-snap/blob/28919a246052d403b865cada03b25f35cc3f6a74/src/k8s/pkg/k8sd/app/cluster_util.go#L35

But it seems unused now. Consider either using the references implementation or remove it as part of this PR please.

Reply:
Right. I've chosen not to use it in this case, because it would return an error if it couldn't stop the k8s-dqlite services (e.g.: if doesn't exist), and leave the control plane services running, and for the remove hook, we should stop any started service regardless.